frontend/javascripts/oxalis/model/bucket_data_handling/data_cube.ts

```diff
@@ -165,12 +156,8 @@ class DataCube {
     this.temporalBucketManager = new TemporalBucketManager(this.pullQueue, this.pushQueue);
   }
 
-  getNullBucket(address: Vector4): Bucket {
-    if (this.boundingBox.containsBucket(address, this.resolutionInfo)) {
-      return NULL_BUCKET;
-    } else {
-      return NULL_BUCKET_OUT_OF_BB;
-    }
```

Comment:
As far as I see, this distinction was only used in the `ArbitraryCubeAdapter`. However, that wasn't used at all, anymore. So, I unified the two `NullBuckets`.